### PR TITLE
Update to align with latest UIs

### PR DESCRIPTION
**Pull request by Nashaath**
## Purpose
Update steps and screenshots to align with latest Choreo UIs.

### CODE REVIEW

File: en/docs/administer/configure-enterprise-login.md [hunk 1]
```diff
@@ -107,31 +107,27 @@ Before you proceed with the configuration, make sure you complete the following:
     3. Click **New Attribute** and select the **Groups** attribute.
     4. Click **Save** and then click **Save Changes**.
 
-### Step 2: Map Asgardeo groups to Choreo roles in the Choreo Console 
+### Step 2: Map Choreo groups to Asgardeo in the Choreo Console
```
Comment (vlgunarathne) [inline en/docs/administer/configure-enterprise-login.md:110]: This should be `Map Choreo groups to enterprise IdP groups in the Choreo Console`. It's not the Asgardeo groups but the enterprise groups that we need to add the mappings.

File: en/docs/administer/configure-enterprise-login.md [hunk 2]
```diff
@@ -107,31 +107,27 @@ Before you proceed with the configuration, make sure you complete the following:
     3. Click **New Attribute** and select the **Groups** attribute.
     4. Click **Save** and then click **Save Changes**.
 
-### Step 2: Map Asgardeo groups to Choreo roles in the Choreo Console 
+### Step 2: Map Choreo groups to Asgardeo in the Choreo Console 
 
 !!! note
     
-    Before you map Asgardeo groups to Choreo roles, ensure you meet the following criteria:
+    Before you map Choreo groups to Asgardeo, ensure you meet the following criteria:
```
Comment (vlgunarathne) [inline en/docs/administer/configure-enterprise-login.md:114]: This too should be enterprise IdP and not Asgardeo

File: en/docs/administer/configure-enterprise-login.md (hunk 1 above)
Comment (Nashaath) [reply en/docs/administer/configure-enterprise-login.md]: Done

File: en/docs/administer/configure-enterprise-login.md (hunk 2 above)
Comment (Nashaath) [reply en/docs/administer/configure-enterprise-login.md]: Done

**Review by vlgunarathne — APPROVED** [review]

**Pull request merged**